docs/index.rst

```diff
@@ -5,65 +5,84 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+Tahoe-LAFS is a Free and Open decentralized cloud storage system.
+It distributes your data across multiple servers.
+Even if some of the servers fail or are taken over by an attacker,
+the entire file store continues to function correctly,
+preserving your privacy and security.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :caption: Preliminaries
 
    about
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Running Tahoe-LAFS
```

Comment:
How about "Using" instead of "Running"?  If I am interested in the software I know I want to use it.  But in addition to runnin it, it seems I also need to install it, configure it, etc.

Reply:
Did this in cf9b388.